pyaedt/generic/settings.py

```diff
@@ -0,0 +1,528 @@
+    #
+    # @is_student.setter
+    # def is_student(self, value):
+    #     self._is_student = value
+
+    @property
+    def desktop_launch_timeout(self):
+        """Set the desktop launcher max timeout. Default is ``90`` seconds.
+
+        Returns
+        -------
+        int
+        """
+        return self._desktop_launch_timeout
+
+    @desktop_launch_timeout.setter
+    def desktop_launch_timeout(self, value):
+        self._desktop_launch_timeout = int(value)
+
+    @property
+    def aedt_environment_variables(self):
+        """Set environment variables to be set before launching a new aedt session.
+        This includes beta features enablemement.
+
+        Returns
+        -------
+        dict
+        """
+        return self._aedt_environment_variables
+
+    @aedt_environment_variables.setter
+    def aedt_environment_variables(self, value):
+        self._aedt_environment_variables = value
+
+    @property
+    def lsf_queue(self):
+        """LSF queue name. This attribute is valid only on Linux
+        systems running LSF Scheduler."""
+        return self._lsf_queue
+
+    @lsf_queue.setter
+    def lsf_queue(self, value):
+        self._lsf_queue = value
+
+    @property
+    def use_lsf_scheduler(self):
+        """Whether to use LSF Scheduler. This attribute is valid only on Linux
+        systems running LSF Scheduler."""
+        return self._use_lsf_scheduler
+
+    @use_lsf_scheduler.setter
+    def use_lsf_scheduler(self, value):
+        self._use_lsf_scheduler = value
+
+    @property
+    def lsf_aedt_command(self):
+        """Get or set the ``ansysedt`` command to launch. The default is ``"ansysedt"``.
+        This attribute is valid only on Linux systems running LSF Scheduler."""
+        return self._lsf_aedt_command
+
+    @lsf_aedt_command.setter
+    def lsf_aedt_command(self, value):
+        self._lsf_aedt_command = value
+
+    @property
+    def lsf_num_cores(self):
+        """Get or set the number of LSF cores. This attribute is valid only
+        on Linux systems running LSF Scheduler."""
+        return self._lsf_num_cores
+
+    @lsf_num_cores.setter
+    def lsf_num_cores(self, value):
+        self._lsf_num_cores = int(value)
+
+    @property
+    def lsf_ram(self):
+        """Get or set the RAM allocated for the LSF job. This attribute is valid
+        only on Linux systems running LSF Scheduler."""
+        return self._lsf_ram
+
+    @lsf_ram.setter
+    def lsf_ram(self, value):
+        self._lsf_ram = int(value)
+
+    @property
+    def lsf_timeout(self):
+        """Get or set the timeout for starting the interactive session. The default is ``3600`` seconds."""
+        return self._lsf_timeout
+
+    @lsf_timeout.setter
+    def lsf_timeout(self, value):
+        self._lsf_timeout = int(value)
+
+    @property
+    def aedt_version(self):
+        """Get and set the aedt version.
+        It disables the sat bounding box for AEDT version > 2022.2.
```

Comment:
```suggestion
        """AEDT version. This attribute disables the sat bounding box for AEDT 2022 R2 and later.
```
Spell out sat. I thought it might be "saturated"--but I wasn't sure.